lib/backend/k8s/k8s_fv_test.go

```diff
@@ -153,6 +153,32 @@ var updateTypeStr = map[api.UpdateType]string{
 	api.UpdateTypeKVUpdated: "Updated",
 }
 
+func (c cb) GetSyncerUpdate(key string) *api.Update {
```

Comment:
This is useful for getting an exact update that was received over the syncer (vs the other existing functions which just verify that an update was recieved).